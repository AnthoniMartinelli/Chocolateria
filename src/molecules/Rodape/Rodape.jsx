import Card from "@material-ui/core/Card";
import CardContent from "@material-ui/core/CardContent";
import React from "react";
import Typography from "@material-ui/core/Typography";
import Grid from "@material-ui/core/Grid";
import { makeStyles } from "@material-ui/core";
import { secondary } from "../../atoms/tema";

const styles = makeStyles(() => ({
  CardStyle: {
    backgroundColor: secondary,
  },
}));

export default function Rodape() {
  const classes = styles();
  return (
    <Card className={classes.CardStyle}>
      <CardContent>
        <Grid container item justify="center">
          <Typography color="primary">
            Chocolates e Cia || {new Date(Date.now()).getFullYear()}
          </Typography>
        </Grid>
      </CardContent>
    </Card>
  );
}
