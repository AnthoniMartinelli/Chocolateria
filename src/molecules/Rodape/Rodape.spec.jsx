import React from "react";
import { cleanup, render } from "@testing-library/react";
import Rodape from "./Rodape";

describe("Rodape", () => {
  afterEach(() => cleanup());
  it("renderiza o Rodape", () => {
    const wrapper = render(<Rodape />);

    expect(wrapper.container).toMatchSnapshot();
  });
});
