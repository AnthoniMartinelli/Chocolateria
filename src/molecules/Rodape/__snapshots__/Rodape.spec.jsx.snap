// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rodape renderiza o Rodape 1`] = `
<div>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-CardStyle-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      class="MuiCardContent-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary"
        >
          Chocolates e Cia || 
          2021
        </p>
      </div>
    </div>
  </div>
</div>
`;
